contrib/sssd.spec.in

```diff
@@ -960,6 +963,7 @@ install -D -p -m 0644 %{SOURCE1} %{buildroot}%{_sysusersdir}/sssd.conf
 %{_datadir}/sssd/krb5-snippets/sssd_enable_passkey
 %{_udevrulesdir}/90-sssd-token-access.rules
 %config(noreplace) %{_sysconfdir}/krb5.conf.d/sssd_enable_passkey
+%{_mandir}/man5/sssd-passkey.5*
```

Comment:
Please add `:packaging:` RN to the commit message.

Reply:
Done.